release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -101,16 +101,47 @@ With this release, the installation program uses Cluster API instead of Terrafor
 [id="ocp-4-17-web-console_{context}"]
 === Web console
 
+[id="ocp-4-17-ocp-lightspeed_{context}"]
+==== {ols} Operator is available in the web console
+
+Starting with {product-title} 4.16, {ols} Operator is available for use in the web console. With this release, a hover button was added to help you discover {ols}. Once you click the hover button, the chat window will appear with instructions on how to enable and install {ols} on the cluster. You can hide the {ols} button when changing the default user preferences.
+
+//https://github.com/openshift/openshift-docs/pull/81581
+//For more information, see xref:../web_console/capabilities_products-web-console.html#openshift-lightspeed-web-console_capabilities-web-console[Enhancing your experience in the {product-title} web console by using {ols-official}].
+
 [id="ocp-4-17-administrator-perspective_{context}"]
 ==== Administrator perspective
 
 This release introduces the following updates to the *Administrator* perspective of the web console:
 
+* Deprecated Operators are displayed in the OperatorHub before and after installation along with a warning notification the Operator is deprecated.
+* Config files for `MachineConfig` objects were moved to a new section enabling you to check the content of the configuation files without having to manually retrieve its contents.
+* An alert was added to the *Operator details* page and the *Operator installation* page if your cluster is on {gcp-first} with Workload Identity Foundation (WIF).
+* A page for ShipWright `BuildStrategy` was added with a `ClusterBuildStrategy` and `BuildStrategy` tab.
+
+[id="ocp-4-17-customize-create-project-modal-dyanmic-plugin"]
+===== Customize the *Create project* modal using dynamic plugins
+
+With this release, a new extension point was added, so dynamic plugin creators can pass a component that renders in place of the default *Create Project* modal.
+
+For more information on {product-title} Console dynamic plugin SDK extensions, see xref:../web_console/dynamic-plugin/dynamic-plugins-reference.adoc#dynamic-plugin-sdk-extensions_dynamic-plugins-reference[Dynamic plugin extension types].
+
+[id="ocp-4-17-console-functional-external-oidc-issuer"]
+===== External OpenID Connect (OIDC) token issuer is now functional in the web console
+
+With this update, the web console works as expected when internal `oauth-server` and `oauth-apiserver` are removed and replaced with an external OpenID Connect (OIDC) issuer.
+
 [id="ocp-4-17-developer-perspective_{context}"]
 ==== Developer Perspective
 
 This release introduces the following updates to the *Developer* perspective of the web console:
 
+* When you use one of the import flows to create a new deployment, the *Import from Git* or *Container images* automatically opens them in the sidebar.
+* Easily select the desired *Git Type* without having to use the list if {product-title} is unable to identify it.
```

Comment:
Make this a complete sentence:
```suggestion
* You can easily select the desired *Git Type* without having to use the list if {product-title} is unable to identify it.
```

Reply:
Also, ambiguous _it_. And will users know what a Git Type is and where to select it?